server/app.js

```diff
@@ -25,8 +27,30 @@ const io = socketio(server, {
   },
 });
 
+io.use(async (socket, next) => {
+  if (socket.handshake.headers.cookie) {
+    token = cookie.parse(socket.handshake.headers.cookie).token;
+    if (token) {
+      const authorized = await jwt.verify(token, process.env.JWT_SECRET);
+      if (authorized.id) {
+        next();
+      } else {
+        next(new Error("Invalid Authorization"));
+      }
+    } else {
+      next(new Error("Authorization Error"));
+    }
+  } else {
+    next(new Error("Authorization Error"));
+  }
+});
+
 io.on("connection", (socket) => {
   console.log("connected");
+
+  socket.on("goOnline", (connectionId) => {
+    console.log(`New socket connection, id:'${connectionId}'`);
```

Comment:
Maybe you need to add this to a list of online users

Reply:
I think it might be okay to just log a user detail for now, based on the wording of the issue #30 - what do you think @ezeilo-su ?

Reply:
Perhaps that would be part of the message sockets ticket?